File: IoTPetFeederDataManager/Models/Feeder.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace IoTPetFeederDataManager.Models
{
    public class Feeder
    {
        [Key]
        public Guid Id { get; set; }
        [Required]
        public string FeederId { get; set; }
        public int FeedOverride { get; set; } = 0;
        public int FeedCount { get; set; } = 0;
        public int CurrentTime { get; set; }
        public int NumberOfFeedTimes { get; set; }
        public string FeedTimes { get; set; }
    }
}

File: PetFeederApp/PetFeederApp/Model/Feeder.swift
//
//  Feeder.swift
//  PetFeederApp
//
//  Created by Jonathan Fletcher on 4/29/20.
//  Copyright © 2020 Jonathan Fletcher. All rights reserved.
//

import Foundation

struct Feeder: Codable {
    
    var id: String
    var feederId: String
    var feedOverride: Int?
    var feedCount: Int?
    var currentTime: Int?
    var numberOfFeedTimes: Int?
    var feedTimes: String?
}
